src/config/rds_production_dependency.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/Originate/exosphere/src/types"
+)
+
+type rdsProductionDependency struct {
+	config    types.ProductionDependencyConfig
+	appConfig types.AppConfig
+}
+
+// GetDockerConfig returns docker configuration and an error if any
+func (r *rdsProductionDependency) GetDockerConfig() (types.DockerConfig, error) {
+	return types.DockerConfig{
+		Image: fmt.Sprintf("%s:%s", r.config.Name, r.config.Version),
```

Comment:
Wondering what we should do here as we still have the un-needed step where a DB dependency's docker image is uploaded to ECR

Reply:
Can we update the ProductionDependency interface to have something like `HasDockerConfig` or something? And the builder doesn't add those when they return false. 

Alternatively, we could make it so a return value from this function of nil means a similar thing. We may need to update the return type to be a pointer to allow nil to be returned

Reply:
What about generic dependencies? They would all either have to be pushed to ECR or not. Should we add yet another field to `ProductionDependencyConfig`?

Reply:
Aren't generic dependencies currently based on a docker image (which means we would always need to push)

Reply:
I'm not even sure what a generic dependency is at this point. I guess we'll just always push for now and then add a field if it becomes necessary in the future.

Reply:
Can this now be updated to return an empty docker config?